cityelf/src/main/java/com/cityelf/repository/UserRepository.java

```diff
@@ -1,36 +1,38 @@
 package com.cityelf.repository;
 
 import com.cityelf.model.User;
+import com.cityelf.model.Notification;
+
 import java.util.ArrayList;
 import java.util.List;
+
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserRepository {
-  private List<User> users = new ArrayList();
-
-  public UserRepository() {
-    User user = new User();
-    user.setFirstname("Oleg");
-    user.setLastname("Mikhailov");
-    user.setAdress("Lenina");
-    user.setEmail("norco_mount@mail.ru");
-    user.setPhone("0936436691");
-
-    users.add(user);
-
-    user = new User();
-    user.setFirstname("Sasha");
-    user.setLastname("Lusenko");
-    user.setAdress("Bunina");
-    user.setEmail("sasha_asdasfas@mail.ru");
-    user.setPhone("063456321");
-
-    users.add(user);
-  }
-
-  public List<User> getUsers()
-  {
-    return users;
-  }
+    private List<User> users = new ArrayList();
```

Comment:
This file hasn't been changed at all. We will add the correct formatting styles.
Just return back 2 spaces indentation as it was for now.
We will discuss that thing. Styling is very important.